l10n_vn_account_invoice_number/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2006-2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Invoice Number',
+    'version': '10.0.1.0.0',
+    'author': 'Trobz,'
+              'Odoo Community Association (OCA)',
+    'website': 'http://trobz.com',
+    'license': 'AGPL-3',
+    'category': 'Accounting & Finance',
+    'depends': [
+        'account',
+        'account_voucher',
+        'sale',
+        'purchase'
```

Comment:
We should remove dependencies to sale, purchase.

I believe that account_voucher can also be removed if we agree that it is not a receipt when there is an invoice number.